cms/static/js/views/course_info_update.js

```diff
@@ -71,7 +71,7 @@ define(["js/views/validation", "codemirror", "js/models/course_update",
         },
 
         handleValidationError : function(model, error) {
-            var ele = this.$el.find('#course-update-list li[name=\"'+model.cid+'\"');
```

Comment:
A `]` was missing. This works on newer browsers but not older ones (including the one that Jenkins uses for running tests).